requirements.txt

```diff
@@ -1,5 +1,5 @@
-PyJWT==1.1.0
-slumber==0.7.1
+# Package requirements
+-e .
```

Comment:
It'd be nice to at least add a comment inline to that effect, in case someone sees this and is confused in the future.